pkgs/applications/editors/jetbrains/default.nix

```diff
@@ -204,18 +204,11 @@ rec {
           (attrs.postFixup or "")
           + lib.optionalString (stdenv.hostPlatform.isLinux) ''
             (
-              cd $out/clion
-
-              # I think the included gdb has a couple of patches, so we patch it instead of replacing
-              ls -d $PWD/bin/gdb/linux/*/lib/python3.8/lib-dynload/* |
-              xargs patchelf \
-                --replace-needed libssl.so.10 libssl.so \
-                --replace-needed libcrypto.so.10 libcrypto.so
-
-              ls -d $PWD/bin/lldb/linux/*/lib/python3.8/lib-dynload/* |
+              find $out -name '*.so' |
```

Comment:
> ```
> patchelf: missing ELF header
> ```

I'm guessing there are some libs that shouldn't be patched, so we need to filter them out somehow?

I wonder if the previous globs for `gdb` and `lldb` are more accurate than finding _all_ `.so` files in `$out`?